test/test_composite.py

```diff
@@ -52,3 +68,290 @@ def test_parallel_options(self):
             self.assertEqual(par_exp.analysis_options, Options())
 
             par_exp.run(FakeBackend())
+
+
+class DummyExperiment(BaseExperiment):
```

Comment:
`FakeExperiment` is `tests.utils` is similar but not exactly what's needed here, for example it assumes a single qubit

Reply:
Can this be merged with `FakeExperiment`? Seems unnecessary to have two classes like this.